### PR TITLE
portfolio PR: Accessibility features 

### PR DESCRIPTION
**Pull request by ngalamac**
In this project,
- We checked the portfolio site for accessibility features and the following report was made:
##REPORT##
- Page Titles: No issues found
- Image text alternatives: We added descriptive image text alternatives. Please refer through the following links 
https://github.com/ishimwepati/Portfolio/blob/4f0c94f34bba962159eb185dc45fa3b52c3423ba/index.html#L43-L47
 https://github.com/ishimwepati/Portfolio/blob/4f0c94f34bba962159eb185dc45fa3b52c3423ba/index.html#L267-L271
https://github.com/ishimwepati/Portfolio/blob/4f0c94f34bba962159eb185dc45fa3b52c3423ba/index.html#L52
https://github.com/ishimwepati/Portfolio/blob/4f0c94f34bba962159eb185dc45fa3b52c3423ba/index.html#L57
- Text headings: We found out the h1 tag was used more than once in thesame html file. We readjusted it by removing one and adjusting its corresponding styling. Please check on this link for reference(
-- html link: https://github.com/ishimwepati/Portfolio/blob/8219163533075e90792f47b5f56a7f31845a0bc3/index.html#L31-L32
https://github.com/ishimwepati/Portfolio/blob/4f0c94f34bba962159eb185dc45fa3b52c3423ba/index.html#L56
https://github.com/ishimwepati/Portfolio/blob/4f0c94f34bba962159eb185dc45fa3b52c3423ba/index.html#L178
----css link: https://github.com/ishimwepati/Portfolio/blob/8219163533075e90792f47b5f56a7f31845a0bc3/style.css#L50-L56
-Color contrast: No issues found
- Resize: No issues found
- Interaction: No issues found
- Moving content: No issues found
- Multimedia: No issues found
- Basic structure of the page: No issues found
We equally added changes requested by previous code reviewer

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -40,21 +40,21 @@ <h1 class="down_word">I’m a software developer</h1>
...
                     <br>
                 </ul>
                 <br>
             </section>
-            <button class="disable_button"><img src="icon/Disabled.png"></button>
+            <button class="disable_button"><img src="icon/Disabled.png" alt="Disabled button"></button>
         </section>
     </section>
     <section class="work">
         <h1 class="upper_word"><br>My Recent <br> Works</h1>
```
Comment (LionRouge1) [inline index.html]: - [ ] Kindly make sure your heading levels have a meaningful hierarchy. Ideally the page starts with an `h1` tag which is usually used only once in a page.
### Your portfolio heading hierarchy from the "About me" section
Screenshot from 2023-06-21 09-12-22

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi **Team**, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good job finding some accessibility issues of your portfolio :clap: 
- Nice work, your report about accessibility of your portfolio is clear and understandable :1st_place_medal: 
- All linters passed :heavy_check_mark: 

## Required Changes ♻️
- [ ] Nice work adding the corresponding line of thing that need to be change in your pull request changes :+1:  However I noticed that you fixed some image text alternatives issues, but it was not include in your report. Kindly add them to your report to follow the project requirement.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -242,10 +242,10 @@ <h1 class="upper_word"><br>About me</h1>
 
     <section id="contact-us" class="contact-us">
                 <div class="word-contact">
-                    <h1 class="upper_word">
+                    <h3 class="upper_word">
                         I'm always interested in hearing about new projects, 
                         so if you'd like to chat please get in touch.
-                    </h1>
+                    </h>
```
Comment (abdulhamiid) [inline index.html]: Please kindly correct the error in the Closing Tag

The opened `<h3>` tag should be closed with `</h3>` 😊👍

Tags usually travel in pairs. An `opening tag` begins a section of page content, and a `closing tag` ends it.

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @ngalamac 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, I really appreciate the effort you put in! but there are still some which haven't been addressed yet.

almost-there

## Highlights 🌟

- ✔️ Descriptions are very descriptive
- ✔️ Use GitHub Flow
- ✔️ Meaningful commit messages.

## Required Changes ♻️
_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by christianbiring1 — APPROVED** [review]
Hi @ishimwepati , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 

## Highlights
- [ ] Great work for implementing the changes as required by the previous reviewer :+1: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**